### ISSUE TITLE
[Bug] Black apple seeds and ink bush seeds crash server after ticking

### ISSUE THREAD

**Issue by greenisacolour** [quiqueck/BetterNether#8]
### What happened?

Every time a Black Apple Seed or Ink Bush Seed is placed then ticked the server crashes. Me and a friend tested every type of seed, BetterNether and BetterEnd, the only ones that crashed the server were the ones listed above

It is reproducible, did the same thing on a local server and singleplayer, same exact problem.
I tried only using BCLib, BetterNether, and Fabric API in singleplayer and it closed the game the exact same way

### BetterNether

7.0.2

### BCLib

2.0.5

### Fabric API

0.56.0

### Fabric Loader

0.14.8

### Minecraft

1.19

### Relevant log output

```shell
[15:55:36] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Exception ticking world
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:879) ~[server-intermediary.jar:?]
	at net.minecraft.class_3176.method_3813(class_3176.java:289) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665) [server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:258) [server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer$$Lambda$5408/0x00000000bafab308.run(Unknown Source) [server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:884) [?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.class_2248.method_9564()" because "this.plant" is null
	at org.betterx.betternether.blocks.BaseBlockCommonSapling.method_9652(BlockCommonSapling.java:97) ~[better-nether-7.0.2.jar:?]
	at org.betterx.betternether.blocks.BlockCommonSapling.method_9652(BlockCommonSapling.java:26) ~[better-nether-7.0.2.jar:?]
	at org.betterx.betternether.blocks.BaseBlockCommonSapling.method_9588(BlockCommonSapling.java:104) ~[better-nether-7.0.2.jar:?]
	at org.betterx.betternether.blocks.BlockCommonSapling.method_9588(BlockCommonSapling.java:26) ~[better-nether-7.0.2.jar:?]
	at net.minecraft.class_4970.method_9514(class_4970.java:366) ~[server-intermediary.jar:?]
	at net.minecraft.class_4970$class_4971.method_26199(class_4970.java:947) ~[server-intermediary.jar:?]
	at net.minecraft.class_3218.redirect$zle000$redirectBlockStateTick(class_3218.java:4231) ~[server-intermediary.jar:?]
	at net.minecraft.class_3218.method_18203(class_3218.java:495) ~[server-intermediary.jar:?]
	at net.minecraft.class_3215.method_14161(class_3215.java:379) ~[server-intermediary.jar:?]
	at net.minecraft.class_3215.method_12127(class_3215.java:323) ~[server-intermediary.jar:?]
	at net.minecraft.class_3218.method_18765(class_3218.java:318) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:875) ~[server-intermediary.jar:?]
	... 6 more
[15:55:36] [Server thread/ERROR]: This crash report has been saved to: /home/container/./crash-reports/crash-2022-06-15_15.55.36-server.txt
[15:55:36] [Server thread/INFO]: Stopping server
[15:55:36] [Server thread/INFO]: Saving players
[15:55:36] [Server thread/INFO]: greenisacolour lost connection: Server closed
[15:55:36] [Server thread/INFO]: greenisacolour left the game
[15:55:36] [Server thread/INFO]: Saving worlds
[15:55:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:overworld
[15:55:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_end
[15:55:38] [Server thread/INFO]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_nether
[15:55:38] [Server thread/INFO]: ThreadedAnvilChunkStorage (world): All chunks are saved
[15:55:38] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[15:55:38] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[15:55:38] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[15:55:38] [Server thread/INFO]: Thread Query Listener stopped
[33m[1m[Pterodactyl Daemon]:[39m ---------- Detected server process in a crashed state! ----------[0m
[33m[1m[Pterodactyl Daemon]:[39m Exit code: 0[0m
[33m[1m[Pterodactyl Daemon]:[39m Out of memory: false[0m
[33m[1m[Pterodactyl Daemon]:[39m Checking server disk space usage, this could take a few seconds...[0m
[33m[1m[Pterodactyl Daemon]:[39m Updating process configuration files...[0m
[33m[1m[Pterodactyl Daemon]:[39m Ensuring file permissions are set correctly, this could take a few seconds...[0m
[33m[1m[Pterodactyl Daemon]:[39m Pulling Docker container image, this could take a few minutes to complete...[0m
[33m[1m[Pterodactyl Daemon]:[39m Finished pulling Docker container image[0m
Java version: 17.0.2
[1;33mNOTE: [0mForge compatibility mode is enabled.
[1;33mNOTE: [0mThe Log4j2 vulnerability workaround has been enabled. If you're running an unpatched server software build, remember to update ASAP as this workaround may be removed at any time, and is not effective in older versions of the game.
[1;33mcustomer@sparkedhost:~$[0m java -Xms128M -Xmx10240M -Dlog4j2.formatMsgNoLookups=true -Dterminal.jline=false -Dterminal.ansi=true -jar fabric-server-mc.1.19-loader.0.14.8-launcher.0.11.0.jar
Starting net.fabricmc.loader.impl.game.minecraft.BundlerClassPathCapture
[15:55:40] [main/INFO]: Loading Minecraft 1.19 with Fabric Loader 0.14.8
[15:55:41] [main/INFO]: Loading 44 mods:
	- bclib 2.0.5
	- betterend 2.0.2
	- betternether 7.0.2
	- fabric 0.56.0+1.19
	- fabric-api-base 0.4.9+e62f51a3a9 via fabric
	- fabric-api-lookup-api-v1 1.6.7+9ff28f40a9 via fabric
	- fabric-biome-api-v1 9.0.13+b2a4a624a9 via fabric
	- fabric-command-api-v1 1.2.5+f71b366fa9 via fabric
	- fabric-command-api-v2 2.1.1+cda9839ba9 via fabric
	- fabric-commands-v0 0.2.22+df3654b3a9 via fabric
	- fabric-containers-v0 0.1.27+df3654b3a9 via fabric
	- fabric-content-registries-v0 3.1.0+4a3ec384a9 via fabric
	- fabric-convention-tags-v1 1.0.8+37622d24a9 via fabric
	- fabric-crash-report-info-v1 0.2.3+bd0a0d4aa9 via fabric
	- fabric-data-generation-api-v1 5.0.1+2a5b9210a9 via fabric
	- fabric-dimensions-v1 2.1.27+72da3b3da9 via fabric
	- fabric-entity-events-v1 1.4.16+9ff28f40a9 via fabric
	- fabric-events-interaction-v0 0.4.26+9ff28f40a9 via fabric
	- fabric-events-lifecycle-v0 0.2.25+df3654b3a9 via fabric
	- fabric-game-rule-api-v1 1.0.19+18990361a9 via fabric
	- fabric-item-api-v1 1.5.4+767db26ea9 via fabric
	- fabric-item-groups-v0 0.3.23+9ff28f40a9 via fabric
	- fabric-lifecycle-events-v1 2.1.0+33fbc738a9 via fabric
	- fabric-loot-api-v2 1.0.1+9e7660c6a9 via fabric
	- fabric-loot-tables-v1 1.1.1+9e7660c6a9 via fabric
	- fabric-message-api-v1 1.0.1+513f4a59a9 via fabric
	- fabric-mining-level-api-v1 2.1.8+33fbc738a9 via fabric
	- fabric-networking-api-v1 1.0.27+7fe97409a9 via fabric
	- fabric-networking-v0 0.3.14+df3654b3a9 via fabric
	- fabric-object-builder-api-v1 4.0.5+9ff28f40a9 via fabric
	- fabric-particles-v1 1.0.8+dc39553aa9 via fabric
	- fabric-registry-sync-v0 0.9.16+92cf9a3ea9 via fabric
	- fabric-rendering-data-attachment-v1 0.3.12+9ff28f40a9 via fabric
	- fabric-rendering-fluids-v1 3.0.5+9ff28f40a9 via fabric
	- fabric-resource-conditions-api-v1 2.0.9+e62f51a3a9 via fabric
	- fabric-resource-loader-v0 0.5.3+9e7660c6a9 via fabric
	- fabric-screen-handler-api-v1 1.2.6+9ff28f40a9 via fabric
	- fabric-transfer-api-v1 2.0.9+e62f51a3a9 via fabric
	- fabric-transitive-access-wideners-v1 1.1.1+9e7660c6a9 via fabric
	- fabricloader 0.14.8
	- java 17
	- lithium 0.8.0
	- minecraft 1.19
	- voicechat 1.19-2.2.45
[15:55:41] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/home/container/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[15:55:41] [main/INFO]: Compatibility level set to JAVA_17
[15:55:41] [main/INFO]: Loaded configuration file for Lithium: 94 options available, 0 override(s) found
[15:55:41] [main/WARN]: Error loading class: net/minecraft/class_5293 (java.lang.ClassNotFoundException: net/minecraft/class_5293)
[15:55:41] [main/WARN]: @Mixin target net.minecraft.class_5293 was not found bclib.mixins.common.json:PresetEditorMixin from mod bclib
[15:55:41] [main/WARN]: Error loading class: net/minecraft/class_7196 (java.lang.ClassNotFoundException: net/minecraft/class_7196)
[15:55:41] [main/WARN]: @Mixin target net.minecraft.class_7196 was not found bclib.mixins.common.json:WorldOpenFlowsMixin from mod bclib
```

### Other Mods

```shell
simplevoicechat 2.2.45
lithium 0.8.0
(main server)
```

**Comment by quiqueck** [comment]
Hi, and thanks a lot for testing this out. I'll fix this for the next release.

**Comment by brynmailer** [comment]
Does anyone have a temporary solution to this until the next release? I'm running a server for a fairly large group of people and it's currently crashing on startup.

**Comment by quiqueck** [comment]
Should be fixed in the just release 7.0.3 Version
